src/Homework 4/longest_substring.py

```diff
@@ -0,0 +1,70 @@
+"""Description:
+Find the longest substring in alphabetical order.
+Example: the longest alphabetical substring
+in "asdfaaaabbbbcttavvfffffdf" is "aaaabbbbctt".
+There are tests with strings up to 10 000 characters long so your
+code will need to be efficient.
+The input will only consist of lowercase characters and will be at
+least one letter long.
+If there are multiple solutions, return the one that appears first.
+Good luck :)
+
+"""
+
+
+def longest(s):
+    start, end = 0, 1
```

Comment:
you solution is completely unclear because of naming :( it's better put talking names than comments what your code do